serverless/channels/connect-channels-event-source.adoc

```diff
@@ -6,7 +6,13 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Connecting a channel to an event source allows the channel to receive events from that source. These events can then be forwarded to an event sink by using subscriptions.
+Connecting a channel to an event source allows the channel to receive events from that source.
 
-include::modules/serverless-connect-channel-source-odc.adoc[leveloffset=+1]
+include::modules/serverless-connect-sink-source-odc.adoc[leveloffset=+1]
 // add kn, create module - modules/serverless-connect-channel-source-kn.adoc
+// Using --sink flag with kn (generic)
+include::modules/specifying-sink-flag-kn.adoc[leveloffset=+1]
+
```

Comment:
Use unique IDs for "Prerequisites", "Additional resources", and "Next steps" titles in assemblies, per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
Oops, somehow missed this one, good call, thanks!